tyrant/constants.py

```diff
@@ -1,6 +1,12 @@
 import os
 from typing import Optional
 
+POSITIVE_VERBS = ["pronounces", "decrees", "proclaims", "ordains"]
+
+NEGATIVE_VERBS = ["commands", "orders", "demands", "dictates"]
+
+UNCERTAIN_VERBS = ["mumbles", "suggests", "mutters", "shrugs, and says"]
```

Comment:
```suggestion
POSITIVE_VERBS = ["pronounces", "decrees", "proclaims", "ordains"]
NEGATIVE_VERBS = ["commands", "orders", "demands", "dictates"]
UNCERTAIN_VERBS = ["mumbles", "suggests", "mutters", "shrugs, and says"]
```

These newlines are noisy. A good rule of thumb is that **similar things belong together**. When we group all the verbs together in one chunk, it improves readability.